File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/while/code-examples/while-if.php
<?php
$items = loadItems();
$index = 0;
while ($index < count($items)) {
    $item = $items[$index];
    if ($item->enabled) {
        processItem($item);
    } else {
        recordSkippedItem($item);
    }
    $index = $index + 1;
}
showSummary();

File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/while/code-examples/while-if.cs
var items = LoadItems(); // List<Item>
var index = 0;
while (index < items.Count) {
    var item = items[index];
    if (item.Enabled) {
        ProcessItem(item);
    } else {
        RecordSkippedItem(item);
    }
    index = index + 1;
}
ShowSummary();
